### PR TITLE
xls to xlsx in grids

### PR DESCRIPTION
**Pull request by Sogl**

### CODE REVIEW

File: Kwf/Controller/Action/Auto/Grid.php [hunk 1]
```diff
@@ -1010,15 +1010,15 @@ public function downloadExportFileAction()
...
             throw new Kwf_Exception('Wrong downloadkey submitted');
         }
         Kwf_Util_TempCleaner::clean();
 
         $file = array(
-            'contents' => file_get_contents('temp/'.$this->_getParam('downloadkey').'.xls'),
-            'mimeType' => 'application/msexcel',
-            'downloadFilename' => 'export_'.date('Ymd-Hi').'.xls'
+            'contents' => file_get_contents('temp/'.$this->_getParam('downloadkey').'.xlsx'),
+            'mimeType' => 'application/octet-stream',
```
Comment (yacon) [inline Kwf/Controller/Action/Auto/Grid.php]: is this the correct mime type for xlsx?
I think it is this one: application/vnd.openxmlformats-officedocument.spreadsheetml.sheet
References:
http://www.sitepoint.com/web-foundations/mime-types-summary-list/
http://stackoverflow.com/questions/4212861/what-is-a-correct-mime-type-for-docx-pptx-etc

**Comment by Sogl** [comment]
That's right. 
However, I am faced with the fact that in different browsers MIME-types differ.

With octet-stream I say to browser: Just save it

**Comment by nsams** [comment]
I'm with @yacon, please use the correct mime type.

The downloadFilename already makes sure the browser offers a file download.

**Comment by Sogl** [comment]
Changed =)

File: Kwf/Controller/Action/Auto/Grid.php [hunk 2]
```diff
@@ -898,7 +898,7 @@ public function downloadCsvExportFileAction()
 
         $file = array(
             'contents' => file_get_contents('temp/'.$this->_getParam('downloadkey').'.csv'),
-            'mimeType' => 'application/octet-stream',
+            'mimeType' => 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
```
Comment (yacon) [inline Kwf/Controller/Action/Auto/Grid.php]: csv? ;)
can you change this in the correct line [1020] please.

**Comment by Sogl** [comment]
Damn. Inattention.

**Comment by nsams** [comment]
merged with ebf5ac1f87988a174a585cf3e4725b2e4dd30200, thanks

(I squashed your commits into a single one)

**Pull request closed without merging**